docs/settings_ref.rst

```diff
@@ -411,3 +411,14 @@ The value of the claim must be a unique value. No 2 users should ever have the s
 .. NOTE::
    You can find the short name for the claims you configure in the ADFS management console underneath
    **ADFS** ➜ **Service** ➜ **Claim Descriptions**
+   
+   
+.. _version_setting:
+
+VERSION
+--------------
+* **Default**: ``v1.0`` 
+* **Type**: ``string``
+
+Version of openid configurations API. By default it is set to 'v1.0'. Developers who wants to use new version can set in their main settings.py file. 
```

Comment:
To my knowledge this isn't OpenID versions. It's a pure Azure thing, so I don't think we should refer to `OpenID` here.

```suggestion
Version of the Azure Active Directory endpoint version. By default it is set to ``v1.0``. At the time of writing this documentation, it can also be set to ``v2.0``. For new projects, ``v2.0`` is recommended. ``v1.0`` is kept as a default for backwards compatibility.
```